.github/workflows/release-python.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: Python package build and publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    container: fedora:latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo dnf -y install dnf-plugins-core
+        sudo dnf -y builddep libcomps.spec
+        pip install --upgrade pip
+        pip install pytest twine scikit-build
+
+    - name: Build Python sdist
+      run: python3 setup.py sdist
+
+    - name: Build manylinux Python wheels
+      uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2014_x86_64
+      with:
+        python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39'
+        build-requirements: 'scikit-build'
+        system-packages: 'make cmake bzip2-devel expat-devel libxml2-devel xz-devel'
+
+    - name: Install and Test Python source package
+      run: |
+        pip install --user dist/*.tar.gz
+        pushd libcomps/src/python/tests/
+        pytest --verbose --color=yes ./
+        popd
+
+    - name: Install and Test Python universal binary wheel package
+      run: |
+        pip install --user dist/libcomps-0.1.15.post1-cp39-cp39-manylinux2014_x86_64.whl
```

Comment:
Hardcoded version of the `.whl` file here.

Reply:
Since we have to manually specify the versions of Python to build wheels for, I figured this was OK, because if Fedora bumps their Python version then we need to edit the file anyway.

However, it wouldn't be ideal to only discover this when a new release is attempted.  Maybe we should freeze the container image to Fedora 33?

Reply:
I'd leave it on fedora:latest.

But apart from the python version (for which I don't have a solution), there's also the libcomps version itself (0.1.15), that's the main problem I was pointing out :slightly_smiling_face:

Reply:
Oh, right.  I was looking at the ```cp39-cp39``` section because I spent half an hour trying to get glob patterns to work.

But a glob pattern for the libcomps version would be fine.

Reply:
@dralley any plans to address this? It's the main outstanding issue with the PR, the rest is mostly nitpicks...

Reply:
Sorry, I've been really busy. I just pushed the commit.

Reply:
Okay, no worries. But can you squash the fix with the commit that it fixes?

Also, looking at this, another question: It only releases the binary package for the python listed here, that is, python 3.9 (inferring from cp39), correct? So this won't work for distros with any other python than what we decide to do the release for?

Reply:
No, it only tests that package, but it releases all of them.

(It's the only one it _can_ test, because you can't use a wheel built for 3.7 with Python 3.9, etc).

Reply:
Oh, right, sorry :slightly_smiling_face: